src/main/test/base/NavigationTest.java

```diff
@@ -0,0 +1,87 @@
+package base;
+
+import org.junit.jupiter.api.Test;
+
+import base.constants.MapConstants;
+
+import base.navigationservice.NavigationService;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NavigationTest {
+
+    @Test
+    public void TestCenter() {
+        //tests the getNextPortalToGetToCenter method
+        //method input - BottomLeftMap 
+        //expected output - BottomCenterMap
+        NavigationService nav = new NavigationService();
```

Comment:
Instead of creating NavigationService in every test - you can create it once as class variable. NavigationService doesn't have any state, so using only one instance for every test will be ok.

Reply:
So how would I do this?

Reply:
```
class NavigationTest {

NavigationService nav = new NavigationService();

@Test
public void TestCenter() {
        String mapName = "BottomCenterMap";
        String expected = MapConstants.BOTTOM_CENTER_MAP;
        String result = nav.getNextPortalToGetToCenter(mapName);
        assertEquals(expected, result);
}

@Test
public void TestCenter() {
        String mapName = "WaterMap";
        String expected = MapConstants.MAIN_MAP;
        String result = nav.getNextPortalToGetToHome(mapName);
        assertEquals(expected, result);
}

// other tests....
```